src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor/DefaultRazorEditorFactoryService.cs

```diff
@@ -88,35 +87,6 @@ public override bool TryGetParser(ITextBuffer textBuffer, [NotNullWhen(returnVal
         return true;
     }
 
-    internal override bool TryGetSmartIndenter(ITextBuffer textBuffer, [NotNullWhen(returnValue: true)] out BraceSmartIndenter? braceSmartIndenter)
```

Comment:
If this method isn't used, do we need the smart indenter at all? Or is this a question you were hoping I could answer? 😛

Reply:
Nothing uses this helper, but there are plenty of manual fetches of the indenter from the property bag.